### ISSUE TITLE
can not reproduce validation accuracy with pretrained model

### ISSUE THREAD

**Issue by lixincn2015** [cvjena/cnn-models#23]
On ImageNet, I only get a top-5 accuracy of 83.86%.
Any idea what's wrong?
